test/src/org/exist/IndexerTest.java

```diff
@@ -202,12 +203,14 @@ private String store_and_retrieve_ws_mixed_content_value(final boolean preserve,
         }
     }
 
+    @Ignore
```

Comment:
Those are new test cases provided by Leif and as the PR heading says, it's an intermediate fix. I thus think it is ok to @ignore tests which still need to be investigated, but keep them for reference.